gensim/utils.py

```diff
@@ -1148,7 +1148,7 @@ def chunkize_serial(iterable, chunksize, as_numpy=False):
         if as_numpy:
             # convert each document to a 2d numpy array (~6x faster when transmitting
             # chunk data over the wire, in Pyro)
-            wrapped_chunk = [[np.array(doc) for doc in itertools.islice(it, int(chunksize))]]
+            wrapped_chunk = [[np.asarray(doc, dtype=dtype) for doc in itertools.islice(it, int(chunksize))]]
```

Comment:
Doesn't this change the existing behaviour? I'm -1 on that, we should keep compatibility. Both in terms of the array `copy` and `dtype`.

Reply:
This is being used as part of LdaState within LdaModel. These function/classes are already parameterized with dtype, so this makes this more consistent with that code base.

Reply:
Sure, being parametrized is fine. It's a good addition.

But changing the existing semantics (potentially not copying arrays, changing float precision) is not fine.

Reply:
Term frequency should always be an integer value, as you can't have half a word in a document. Not sure why mmcorpus is returning documents as (int,float) while bow returns (int,int)

In both cases this is creating a numpy array, for use in serializing in a distributed environment, which means a copy of the array is being received on the lda_worker process.

Reading asarray documentation,  `copy=False` is being ignored as there is datatype change being made between the source and destination array.

If it makes it clearer, I can change this to 
```
np.array(doc, dtype=dtype)
```

Reply:
@darindf yes, this will be better, I agree